api/src/main/java/com/codeforcommunity/api-spec.md

```diff
@@ -246,6 +246,61 @@ Gets the information for this specific team. Including the members with how many
 }
 ```
 
+## `GET /teams/user_teams`
+
+Gets the teams and teaminfo that the current user is on
+
+## Responses
+
+##### `200 OK`
+
+```json
+{
+    "teams": [
+        {
+            "id": 2,
```

Comment:
Notice how every other route definition doesn't just copy-paste a body and instead gives the types of the JSON data. This is sloppy, take this seriously.